app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt

```diff
@@ -207,6 +207,14 @@ class AccountsFragment : BaseFragment(), OnRefreshListener, AccountsView, Recycl
         currentFilterList = null
     }
 
+    /**
+     * This method is called when the fragment is visible to the user and actively running.
+     **/
+    override fun onResume() {
+        onRefresh()
+        super.onResume()
```

Comment:
Is there any specific reason we are called `super.onResume` after `onRefresh` ? if no let's reverse the order of these calls

Reply:
Done!